src/Tickets/Seating/Service/Maps.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * The service component used to fetch the Maps from the service.
+ *
+ * @since TBD
+ *
+ * @package TEC\Controller\Service;
+ */
+
+namespace TEC\Tickets\Seating\Service;
+
+use TEC\Tickets\Seating\Tables\Maps as Maps_Table;
+
+class Maps {
+	public static function fetch_all() {
```

Comment:
This method is not actually pulling maps from the service, it's just pulling them from the database, look at [the `Service\Layouts::get_in_option_format` method for an example.](https://github.com/the-events-calendar/event-tickets/blob/9afd3bb8e4a87f1bddd75694cd46f2e291346dc9/src/Tickets/Seating/Service/Layouts.php#L100)